project/UnitTests/Core/Config/PreprocessorTest.cs

```diff
@@ -13,7 +13,7 @@ namespace ThoughtWorks.CruiseControl.UnitTests.Core.Config
     [TestFixture]
     public class PreprocessorTest
     {
-    	private static readonly string FAKE_ROOT = Path.DirectorySeparatorChar == '\\' ? "c:\\temp folder\\" : "/tmp/";
+    	private static readonly string FAKE_ROOT = Platform.IsWindows ? "c:\\temp folder\\" : "/tmp/";
```

Comment:
Can't we use [Path.GetTempPath](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.gettemppath?view=netframework-4.7&tabs=linux) here? Or the env. variables (see the remarks section in the link)?

Reply:
Ah but we could, it's just that I did the very minimal set of changes for it to work. I mean, the code was there long before my changes, I simply replaced the test on `DirectorySeparatorChar` with a call to a property with a clearer name.

Reply:
Created #307 for this enhancement